### PR TITLE
Update Team Members and Communication channels

### PR DESCRIPTION
**Pull request by gnzlbg**

### CODE REVIEW

**Review by denzp — APPROVED** [review]
LGTM!

**Review by bheisler — APPROVED** [review]

**Comment by bheisler** [comment]
The community team asked that we not call ourselves a "working group" until they can make it official.

Should we maybe change it to say "interest group" or "unofficial working group" or something like that?

File: README.md [hunk 1]
```diff
@@ -7,11 +7,18 @@ Group](https://github.com/rust-cuda/wg/issues/1).
...
+* @bheisler
+* @denzp
+* @gnzlbg
+* @termoshtt
+* @vadixidav
 
-* https://rust-lang.zulipchat.com/
+**Contact:**
+
+* Chat: https://rust-cuda.zulipchat.com
```
Comment (termoshtt) [inline README.md:20]: I cannot join this zulip room like https://github.com/rust-cuda/wg/issues/3#issuecomment-437951188.

**Review by termoshtt — APPROVED** [review]
Send me Zulip inivitation

**Comment by gnzlbg** [comment]
I would just call us a “team” instead of a “working group”.  Feel free to
open a PR :)

**Comment by gnzlbg** [comment]
@termoshtt it should work without an invitation already :)

**Pull request merged**